docs/modules/getting_started/pages/importing.adoc

```diff
@@ -24,13 +24,16 @@ To import an existing dashboard, follow these steps:
 +
 image:gf-community-dash.png[]
 
-. In {page-component-title}, click on the plus sign in the left menu and choose *Import*.
-. In the *Grafana.com Dashboard* field, enter the dashboard ID and click *Load*. 
-.. The Import screen appears:
-
+. In {page-component-title}, click the plus sign in the left menu and choose *Import*.
+. In the *Import via grafana.com* field, enter the dashboard ID and click *Load*. 
 +
-image:gf-dash-import.png[]
+The Import screen appears.
 
 . Change information as desired and click *Import*. 
-. Go to *Dashboards>Manage* to view/edit the imported dashboard. 
++
+You must specify a flow and performance data source. 
```

Comment:
Depending on the datasource types used in the dashboard, you will be prompted to select the corresponding datasource(s) in your Grafana instance.